src/content/contribute/writing-a-loader.md

```diff
@@ -130,6 +121,31 @@ Take the case of rendering a template file with data specified via loader option
 
 T> The fact that loaders can be chained also means they don't necessarily have to output JavaScript. As long as the next loader in the chain can handle its output, the loader can return any type of module.
 
+### Data sharing
+
+As discussed earlier, webpack loaders can be chained. So, along with passing content to the next chained loader, we can even share data with the next loader. There are two ways to share data among loaders.
```

Comment:
> So, along with passing content to the next chained loader, we can even share data with the next loader.

This oration says basically the same thing. It is there an important difference?

Reply:
actually content here is the source code but we can we share other data as well to next loader using 
`this.callback`

the one with pitch method is not correct as it can only pass data from pitch phase to normal phase of the same loaders.

I will remove the pitch example